docs/dev_guide/2.quick_start.md

```diff
@@ -0,0 +1,160 @@
+# Quick Start
+
+This documentation is *NOT* intended to be comprehensive; it is meant to be a quick guide for the most useful things. For more information, see the develop guide in its entirety.
+
+## Asking Questions
+
+Before asking a question, make sure you have:
+
+- Searched open and closed:
+  - [KCLVM GitHub Issues](https://github.com/KusionStack/KCLVM/issues?utf8=%E2%9C%93&q=is%3Aissue)
+
+- Read the documentations:
+  - [KCLVM Documents](https://kusionstack.io/docs/reference/lang/lang/tour)
+  - [KCLVM Readme](https://github.com/KusionStack/KCLVM)
+
+If you have any questions about `KCLVM`, you are welcome to ask your questions in [KCLVM Github Issues](https://github.com/KusionStack/KCLVM/issues). When you ask a question, please describe the details as clearly as possible so that others in the KCLVM community can understand, and you *MUST* be polite and avoid personal attack and avoid not objective comparison with other projects.
+
+## Cloning and Building `KCLVM`
+
+### Dependencies
+
+#### macOS and OS X
+
+- `git`
+- `Python 3.7+`
+- `Rust 1.60+`
+- `openssl@1.1`
+- `LLVM 12`
+
+You can install `openssl@1.1` using `brew`:
```

Comment:
remove openssl install cause build-in py is removed.

Reply:
> remove openssl install cause build-in py is removed.

removed